templates/admin_transfers_section.php

```diff
@@ -66,7 +67,11 @@
     // FIXME: move the code away from wanting to know the total.
     //       if the user has 1000 tuples do we really want to show 1000/15 direct page links
     //       or should we instead allow queries on timeframe etc.
-    $total_count = 100;
+    $total_count = Transfer::count(array(
```

Comment:
I would also like to short this out if `$offset` is zero and there are no 'real' conditions in the where clause. There are three tables shown by default and calculating the exact total_count for those cases might be quite expensive on a very large database. Especially since one condition starts simply as status='available' which will likely be quite a number of tuples. I have an update to do this so admins with large databases might see the page right away at the expense of maybe seeing page 2, 3 etc for closed tables.